eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/UserConverter.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter;
+
+use eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter;
+use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldDefinition;
+use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldValue;
+use eZ\Publish\SPI\Persistence\Content\FieldValue;
+use eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition;
+
+class UserConverter implements Converter
+{
+    private const PASSWORD_VALIDATOR_IDENTIFIER = 'PasswordValueValidator';
+
+    private const REQUIRE_AT_LEAST_ONE_UPPER_CASE_CHAR = 1;
+    private const REQUIRE_AT_LEAST_ONE_LOWER_CASE_CHAR = 2;
+    private const REQUIRE_AT_LEAST_ONE_NUMERIC_CHAR = 4;
+    private const REQUIRE_AT_LEAST_ONE_NON_ALPHANUMERIC_CHAR = 8;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function toStorageValue(FieldValue $value, StorageFieldValue $storageFieldValue): void
+    {
+        // There is no contained data. All data is external. So we just do
```

Comment:
nitpick: I think it will be ok to have this here and in method below in one line